docs/userguide/data_buildtool.md

```diff
@@ -0,0 +1,380 @@
+        categories: [
+          unit_tree
+        ]
+        rules: [
+          -/units
+          -/unitsNarrow
+          -/unitsShort
+          +/*/length
+        ]
+      }
+    ]
+
+At this time, the wildcard character must stand alone in a piece of the path.
+
+#### Resource Filter for Specific File
+
+The resource filter object takes an optional *files* setting which accepts a file filter in the same syntax used above for file filtering.  For example, if you wanted to apply a filter to misc/supplementalData.txt, you could do the following (this example removes calendar data):
+
+    resourceFilters: [
+      {
+        categories: ["misc"]
+        files: {
+          whitelist: ["supplementalData"]
+        }
+        rules: [
+          -/calendarData
+        ]
+      }
+    ]
+
+#### Combining Multiple Resource Filter Specs
+
+You can also list multiple resource filter objects in the *resourceFilters* array; the filters are added from top to bottom.  For example, here is an advanced configuration that includes "mile" for en-US and "kilometer" for en-CA; this also makes use of the *files* option:
+
+    resourceFilters: [
+      {
+        categories: ["unit_tree"]
+        rules: [
+          -/units
+          -/unitsNarrow
+          -/unitsShort
+        ]
+      },
+      {
+        categories: ["unit_tree"]
+        files: {
+          filterType: locale
+          whitelist: ["en_US"]
+        }
+        rules: [
+          +/*/length/mile
+        ]
+      },
+      {
+        categories: ["unit_tree"]
+        files: {
+          filterType: locale
+          whitelist: ["en_CA"]
+        }
+        rules: [
+          +/*/length/kilometer
+        ]
+      }
+    ]
+
+The above example would give en-US these resource filter rules:
+
+    -/units
+    -/unitsNarrow
+    -/unitsShort
+    +/*/length/mile
+
+and en-CA these resource filter rules:
+
+    -/units
+    -/unitsNarrow
+    -/unitsShort
+    +/*/length/kilometer
+
+In accordance with *filterType* "locale", the parent locales *en* and *root* would get both units; this is required since both en-US and en-CA may inherit from the parent locale:
+
+    -/units
+    -/unitsNarrow
+    -/unitsShort
+    +/*/length/mile
+    +/*/length/kilometer
+
+## Other Features of ICU Data Build Tool
+
+While data filtering is the primary reason the ICU Data Build Tool was developed, there are other features you can use it for, too.
+
+### Running Data Build without Configure/Make
+
+You can build the dat file outside of the ICU build system by directly invoking the Python buildtool.  Run the following command to see the help text for the CLI tool:
+
+    $ PYTHONPATH=path/to/icu4c/source/data python3 -m buildtool --help
+
+### Collation UCAData
+
+There are two versions of the Collation data file:
+
+- ucadata-unihan.txt
+- ucadata-implicithan.txt
+
+By default, the unihan version is used.  To use the implicithan version, pass the following command line argument to buildtool:
```

Comment:
We need to say what the differences are and why someone might want to choose one vs. the other.

How about

The unihan version sorts Han characters in radical-stroke order according to Unicode, which is a somewhat useful default sort order, especially for use with non-CJK languages. The implicithan version sorts Han characters in the order of their Unicode assignment, which is similar to radical-stroke order for common characters but arbitrary for others. The unihan version of the data file is much larger than that for implicithan, so if you need collation but also small data, then you may want to select the implicithan version.

Include sample data file sizes for the two versions in ICU 64.
Include link: https://www.unicode.org/reports/tr10/#Implicit_Weights

Reply:
Thank you for this!